tests/Twig/Extension/CanonicalizeExtensionTest.php

```diff
@@ -242,6 +242,9 @@ public function select2LocalesProvider()
 
     private function changeLocale(string $locale): void
     {
-        $this->requestStack->getCurrentRequest()->setLocale($locale);
+        $request = $this->requestStack->getCurrentRequest();
+        \assert(null !== $request);
```

Comment:
we can use an if here instead, right?

Reply:
We set a request in the requestStack in the setUp, so I think it's ok to assert instead of a dead if.

Reply:
We could change `$request` for `$requestStack` as property of the test on `setUp` like:

```php
$this->request = new Request();
$requestStack = new RequestStack();
$requestStack->push($this->request));
$this->twigExtension = new CanonicalizeExtension($requestStack);
```

Reply:
Done